src/PostgREST/ApiRequest/Preferences.hs

```diff
@@ -41,7 +41,7 @@ import Protolude
 data Preferences
   = Preferences
     { preferResolution     :: Maybe PreferResolution
-    , preferRepresentation :: PreferRepresentation
+    , preferRepresentation :: Maybe PreferRepresentation
```

Comment:
This codecov warning can be ignored. It's a limitation of our tooling right now.